src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SingleSpaceSeparatorCheck.java

```diff
@@ -77,6 +85,17 @@
  *   &lt;property name=&quot;validateComments&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int foo()   { &#47;&#47;Violation, 3 whitespaces
+ * return 1+2  ; } &#47;&#47;Violation, 2 whitespaces
+ * void fun1() {    &#47;&#47;Violation, 4 whitespaces
+ *   return ; } &#47;&#47;OK
+ * void  fun2() {} &#47;&#47;Violation, 2 whitespaces
+ * void fun3() {  &#47;*Violation here, 2 whitespaces
```

Comment:
should be line comment

Reply:
But the example shown in the description also illustrates single line comments, so should i remove them?
Also there is a multi-line comment from line number 95 till line number 96. Another one on line 97.